### PR TITLE
Add workarounds and changes to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lukebiddell**
My first pull request, let me know if there's any issues with it.

----
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -76,12 +76,18 @@ Following a small guidance how we suggest to start.
    We suggest [IntelliJ Community Edition](https://www.jetbrains.com/idea/download/)
    and install the following plugins:
    - EditorConfig by JetBrains (might be already activated)
-   - Spek Framework by Spek Team 
+   - Spek by Ranie Jade Ramiso
```
Comment (robstoll) [inline .github/CONTRIBUTING.md]: That's the plugin for spek 1.x and usually new contributors should only use spek 2.x 
Thus I am not sure if it makes sense to mention it as well but if you think yes, then please add both as follows:
```suggestion
   - Spek Framework by Spek Team (for Spek 2.x)
   - Spek by Ranie Jade Ramiso (for Spek 1.x)
```

Comment (lukebiddell) [reply .github/CONTRIBUTING.md]: Strangely I couldn't find Spek Framework in the plugin market when I originally searched for it, but I see it now. So I think we should leave it as it was before?

Comment (robstoll) [reply .github/CONTRIBUTING.md]: Most likely because spek wasn't yet (or still isn't) available for intellij 2019.3

**Comment by robstoll** [comment]
@lukebiddell thanks for the improvement :+1:

**Pull request merged**